### DISCUSSION TITLE
Where to locate log files?

### DISCUSSION THREAD

**Discussion by yii-joblo** [directus/directus#3864] [Archive]
A there any log files?
Where can I see when for example something is crashing etc?

**Comment by benhaynes** [comment]
@rijkvanzanten @philleepflorence — another good one for Docs! :)

**Reply by philleepflorence** [reply]
Awaiting clarification from @rijkvanzanten ... assuming logs for the App and API.

**Reply by rijkvanzanten** [reply]
Everything is logged into terminal when you run Directus

**Reply by glowkeeper** [reply]
> Everything is logged into terminal when you run Directus

When you say _everything_, exactly what is that? Express API calls? the directus activity table? And is there a way of seeing that terminal output in the admin app'? If not, what do I have to do to make that so?

**Reply by glowkeeper** [reply]
Bump :)

**Reply by rijkvanzanten** [reply]
express-pino (so access logs), and errors in case of unexpected failures. You can use `LOG_LEVEL` to control what shows up  (eg error/warn/info/etc).

> And is there a way of seeing that terminal output in the admin app'?

Not natively in Directus, no. It's sent to STDOUT of whatever server you're using. 

> If not, what do I have to do to make that so? 

Very good question! I suppose you could write the output to a file by using `LOG_STYLE` `raw` (https://docs.directus.io/reference/environment-variables/#general) and piping the output to `pino-tee` (eg `npx directus start | pino-tee`) https://github.com/pinojs/pino-tee, and then go from there. That being said, most accountability info is already available in `/activity` of the app, so you might not need the full API access log streamed to somewhere. (feel free to open a feature request to have a way to stream those back-end-logs into the app btw 👍🏻 )
